homeassistant/components/homeassistant/__init__.py

```diff
@@ -205,7 +205,7 @@ async def async_handle_update_service(call: ha.ServiceCall) -> None:
         ]
 
         if tasks:
-            await asyncio.wait(tasks)
+            await asyncio.gather(*tasks)
```

Comment:
This is a behavior change since if one throws and exception it will get raised and continue on instead of waiting.

Reply:
True. But we do want they exception to bubble up. Right now they are hidden and lost in space.